Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY: ALL venv roles production letsencrypt retry clean clean-all macos-keybase tf-init tf-plan tf-apply check-rtk-prod check-rtk-staging check-planningalerts apply-rtk-prod apply-rtk-staging apply-planningalerts update-github-ssh-keys
 ALL: roles .vagrant
-PRODUCTION := .keybase roles
+KEYSANDROLES := .keybase roles
```

Comment:
I renamed this because I wasn't sure what 'production' meant here.
@jamezpolley does this seem okay to you?